source/extensions/tracers/opencensus/opencensus_tracer_impl.cc

```diff
@@ -28,6 +28,8 @@ namespace Extensions {
 namespace Tracers {
 namespace OpenCensus {
 
+constexpr char kGoogleStackdriverTraceAddress[] = "cloudtrace.googleapis.com";
```

Comment:
nit: Envoy style does not use `kConstant`, just `Constant` or `CONSTANT`.

Reply:
Done.